apps/scan/backend/src/store.ts

```diff
@@ -657,16 +657,6 @@ export class Store {
     return scannerBackedUpAt >= DateTime.max(...cvrsLastUpdatedDates);
   }
 
-  addBallotCard(batchId: string): string {
```

Comment:
This method was unused and `ballot_cards` doesn't exist in the schema. I deleted it to because test coverage was under threshold by 0.01% even though I didn't make any changes to /scan. Deleting these lines puts this PR back over threshold